geomstats/geometry/connection.py

```diff
@@ -14,12 +14,18 @@
 
 
 class Connection:
-    """Class for affine connections.
+    r"""Class for affine connections.
 
     Parameters
     ----------
-    dim: int
+    dim : int
         Dimension of the underlying manifold.
+    default_point_type : str, {\'vector\', \'matrix\'}
+        Point type.
+        Optional, default: \'vector\'.
```

Comment:
ditto

Reply:
I'd rather stay with the more explicit convention if that's ok?